services/cinema_hall.py

```diff
@@ -0,0 +1,16 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:
```

Comment:
The function `get_cinema_halls` is currently annotated to return `None`, but it actually returns a `QuerySet` of `CinemaHall` objects. Consider changing the return type annotation to `QuerySet[CinemaHall]`.